can't compile with cmake

<!--

NOTE: this form is for bug reports only.
For questions or troubleshooting, please post on the protobuf mailing list:
https://groups.google.com/forum/#!forum/protobuf 
Stack Overflow is also a useful if unofficial resource https://stackoverflow.com/questions/tagged/protocol-buffers
-->

**What version of protobuf and what language are you using?**
Version: main/v3.21.1
Language: C++

**What operating system (Linux, Windows, ...) and version?**
Linux
**What runtime / compiler are you using (e.g., python version or gcc version)**
cmake
**What did you do?**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**What did you expect to see**
successful compilation
**What did you see instead?**
error
Make sure you include information that can help us debug (full error message, exception listing, stack trace, logs).

**Anything else we should know about your project / environment**

Scanning dependencies of target protobuf-test-common
[ 62%] Building CXX object CMakeFiles/protobuf-test-common.dir/src/google/protobuf/arena_test_util.cc.o
[ 62%] Building CXX object CMakeFiles/protobuf-test-common.dir/src/google/protobuf/map_lite_test_util.cc.o
In file included from /space/s1/lenore/software/protobuf_again/protobuf/src/google/protobuf/map_lite_test_util.cc:31:
/space/s1/lenore/software/protobuf_again/protobuf/src/google/protobuf/map_lite_test_util.h:34:10: fatal error: google/protobuf/map_lite_unittest.pb.h: No such file or directory
   34 | #include <google/protobuf/map_lite_unittest.pb.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/protobuf-test-common.dir/build.make:301: CMakeFiles/protobuf-test-common.dir/src/google/protobuf/map_lite_test_util.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:153: CMakeFiles/protobuf-test-common.dir/all] Error 2
make: *** [Makefile:141: all] Error 2

I'm still getting the same error
```
Scanning dependencies of target protobuf-test-common
[ 62%] Building CXX object CMakeFiles/protobuf-test-common.dir/src/google/protobuf/arena_test_util.cc.o
[ 62%] Building CXX object CMakeFiles/protobuf-test-common.dir/src/google/protobuf/map_lite_test_util.cc.o
In file included from /space/s1/lenore/software/protobuf_again2/protobuf/src/google/protobuf/map_lite_test_util.cc:31:
/space/s1/lenore/software/protobuf_again2/protobuf/src/google/protobuf/map_lite_test_util.h:34:10: fatal error: google/protobuf/map_lite_unittest.pb.h: No such file or directory
   34 | #include <google/protobuf/map_lite_unittest.pb.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/protobuf-test-common.dir/build.make:301: CMakeFiles/protobuf-test-common.dir/src/google/protobuf/map_lite_test_util.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:153: CMakeFiles/protobuf-test-common.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

--- a/cmake/tests.cmake
+++ b/cmake/tests.cmake
@@ -81,7 +81,6 @@ add_library(protobuf-lite-test-common STATIC
 target_link_libraries(protobuf-lite-test-common ${protobuf_LIB_PROTOBUF_LITE} GTest::gmock)
 
 set(common_test_files
-  ${lite_test_util_srcs}
   ${test_util_hdrs}
   ${test_util_srcs}
   ${mock_code_generator_srcs}
@@ -150,13 +149,11 @@ add_test(NAME lite-test
 
 add_custom_target(check
   COMMAND tests
-  COMMAND lite-test
   DEPENDS tests lite-test test_plugin
   WORKING_DIRECTORY ${protobuf_SOURCE_DIR})
 
 add_test(NAME check
-  COMMAND tests ${protobuf_GTEST_ARGS}
-  WORKING_DIRECTORY "${protobuf_SOURCE_DIR}")
+  COMMAND tests ${protobuf_GTEST_ARGS})
 
 # For test purposes, remove headers that should already be installed.  This
 # prevents accidental conflicts and also version skew (since local headers take

